src/models/organizations.rs

```diff
@@ -0,0 +1,87 @@
+use super::{
+    ids::{Base62Id, TeamId},
+    projects::DonationLink,
+    teams::{ProjectPermissions, TeamMember},
+};
+use serde::{Deserialize, Serialize};
+
+/// The ID of a team
+#[derive(Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(from = "Base62Id")]
+#[serde(into = "Base62Id")]
+pub struct OrganizationId(pub u64);
+
+/// An organization of users who control a project
+#[derive(Serialize, Deserialize)]
+pub struct Organization {
+    /// The id of the organization
+    pub id: OrganizationId,
+    /// The slug of the organization
+    pub slug: String,
+    /// The associated team of the organization
+    pub team_id: TeamId,
+    /// The name of the organization
+    pub name: String,
+    /// The description of the organization
+    pub description: String,
+    /// The website url of the organization
+    pub website_url: Option<String>,
+    /// The discord url of the organization
+    pub discord_url: Option<String>,
```

Comment:
I think we should restructure this to an HashMap of an enum and string. this is the plan for other link structures, but since this is new we should do it here